api/openshift/jobs/integrations.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1beta1
```

Comment:
This is outdated.  

Could remove it even now that we aren't focusing on scheduled jobs anymore.